gems/pending/spec/appliance_console/database_replication_standby_spec.rb

```diff
@@ -0,0 +1,218 @@
+      expect(subject).to receive(:data_dir_empty?).and_return(true)
+      expect(subject).to receive(:generate_cluster_name).and_return(true)
+      expect(subject).to receive(:create_config_file).and_return(true)
+      expect(subject).to receive(:clone_standby_server).and_return(false)
+      expect(subject).to_not receive(:start_postgres)
+      expect(subject).to_not receive(:register_standby_server)
+      expect(subject.activate).to be false
+    end
+
+    it "returns false when start_postgres fails" do
+      expect(subject).to receive(:data_dir_empty?).and_return(true)
+      expect(subject).to receive(:generate_cluster_name).and_return(true)
+      expect(subject).to receive(:create_config_file).and_return(true)
+      expect(subject).to receive(:clone_standby_server).and_return(true)
+      expect(subject).to receive(:start_postgres).and_return(false)
+      expect(subject).to_not receive(:register_standby_server)
+      expect(subject.activate).to be false
+    end
+
+    it "returns false when register_standby_server fails" do
+      expect(subject).to receive(:data_dir_empty?).and_return(true)
+      expect(subject).to receive(:generate_cluster_name).and_return(true)
+      expect(subject).to receive(:create_config_file).and_return(true)
+      expect(subject).to receive(:clone_standby_server).and_return(true)
+      expect(subject).to receive(:start_postgres).and_return(true)
+      expect(subject).to receive(:register_standby_server).and_return(false)
+      expect(subject.activate).to be false
+    end
+  end
+
+  context "create standby server" do
+    before do
+      allow(Process::UID).to receive(:change_privilege)
+      allow(Process::UID).to receive(:from_name)
+    end
+
+    context "#clone_standby_server" do
+      it "raises an error when REGISTER_CMD fails" do
+        result = double(:output => "", :error => "")
+        allow(AwesomeSpawn).to receive(:run!).and_raise(AwesomeSpawn::CommandResultError.new("", result))
+        expect(subject.clone_standby_server).to be false
+      end
+    end
+
+    context "#register_standby_server" do
+      it "raises an error when REGISTER_CMD fails" do
+        result = double(:output => "", :error => "")
+        allow(AwesomeSpawn).to receive(:run!).and_raise(AwesomeSpawn::CommandResultError.new("", result))
+        expect(subject.register_standby_server).to be false
+      end
+
+      it "Succeed when REGISTER_CMD succeeds" do
+        stub_const("ApplianceConsole::DatabaseReplicationStandby::REGISTER_CMD", "pwd")
+        expect(subject.register_standby_server).to be true
+      end
+    end
+  end
+
+  context "#ask_for_standby_host" do
+    it "should use default prompts" do
+      subject.standby_host = "defaultstandby"
+      expect(subject)
+        .to receive(:ask_for_ip_or_hostname).with(/^standby.*hostname/i, "defaultstandby").and_return("defaultstandby")
+      subject.ask_for_standby_host
+    end
+
+    it "should return without default prompts" do
```

Comment:
What is this testing? If we're testing the default value, wouldn't it make more sense to test that it was set to the ip address we stubbed out up here? https://github.com/ManageIQ/manageiq/pull/9585/files#diff-4a1302871566acee10082460b4518624R13